code/lib/cli/src/add.ts

```diff
@@ -23,6 +22,8 @@ const postinstallAddon = async (addonName: string, isOfficialAddon: boolean) =>
     LEGACY_CONFIGS.forEach((config) => {
       try {
         const codemod = require.resolve(
+          // TODO: fix this before merging
+          // @ts-expect-error getPackageName undefined, so will always go to catch branch
           `${getPackageName(addonName, isOfficialAddon)}/postinstall/${config}.js`
```

Comment:
I don't understand this, seems impossible for this code to work if getPackageName is not imported